ios/swift/AmpliSwiftSampleApp/Shared/Ampli/Ampli.swift

```diff
@@ -0,0 +1,766 @@
+/**
+ * Ampli - A strong typed wrapper for your Analytics
+ *
+ * This file is generated by Amplitude.
+ * To update run 'ampli pull swift-ampli'
+ *
+ * Required dependencies: Amplitude-iOS^8.6.0
+ * Tracking Plan Version: 0
+ * Build: 1.0.0
+ *
+ * [View Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest)
+ *
+ * [Full Setup Instructions](https://data.amplitude.com/test-codegen/Test%20Codegen/implementation/swift-ampli)
+ */
+
+import Foundation
+import Amplitude
+
+public typealias MiddlewareExtra = [String: Any]
+
+public enum AmpliEnvironment: Int {
+  case development,production;
+}
+
+let ApiKey: [AmpliEnvironment: String] = [
+  .development:"00aa083ba31d20782808820370c15a71",
```

Comment:
We should not commit any API keys to this this (public) repo. Can you please remove these from the source code and load from external configuration like in the TS examples (.env, .env.example)

Reply:
yeah, I updated to use environment inside xcode schema, let me know if this is best or we prefer other ways.

Reply:
Thanks I think that works. Can you please also add a README to the sample that tell users how to set the external API keys along with `pod install` open `xcworkspace` instructions for newbies.

Reply:
Sounds good, let me add one README